source/docs/prismic/installation.md

```diff
@@ -29,6 +29,10 @@ FRONT_COMMERCE_PRISMIC_WEBHOOK_SECRET=a-secret-defined-in-webhook-configuration
 * `FRONT_COMMERCE_PRISMIC_ACCESS_TOKEN` is the access token for the repository, go to _Settings > API & Security_ and create an application and copy the _Permanent access token_ generated for this application
 * `FRONT_COMMERCE_PRISMIC_WEBHOOK_SECRET` is a secret key used to clear caches in Front-Commerce. To define it, go to _Settings > Webhook_ and create a webhook pointing to your Front-Commerce URL `https://my-shop.example.com/prismic/webhook`. In the webhook form, you can configure a _Secret_. This is the one you should use in this environment variable.
 
+<blockquote class="tip">
+In case of trouble, `front-commerce:prismic` can be added to the `DEBUG` environment variable value, this value turns the debug on for Prismic module and make it verbose.
```

Comment:
Could you please add it to the https://developers.front-commerce.com/docs/reference/environment-variables.html#Debugging page?

As part of #212, I only grepped the FC core repository (not the other projects) and didn't find this one.

Reply:
fixed in 50d1dae